nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/ui/web/auth/service/PluggableAuthenticationService.java

```diff
@@ -97,20 +96,20 @@
 
     private List<String> authChain;
 
-    private final Map<String, SpecificAuthChainDescriptor> specificAuthChains = new HashMap<String, SpecificAuthChainDescriptor>();
+    private final Map<String, SpecificAuthChainDescriptor> specificAuthChains = new HashMap<>();
```

Comment:
You shouldn't cleanup in a commit with other kinds of changes, it makes it harder to review.